### ISSUE TITLE
How to generate parse strings from existing DataFrame?

### ISSUE THREAD

**Issue by xiaodaigh** [tk3369/CSVReader.jl#2]
I've read in the data using 

```julia
a = CSVReader.read_csv("c:/data/Performance_All/Performance_2016Q1.txt"; delimiter = '|', headers = false)
```

and now I want to save down the parser string, so that it can load faster. `a` has thousands of columns and I don't want to write it by hand. what's the best way to achieve it?

**Comment by tk3369** [comment]
If your column types repeat, then even writing it by hand is not difficult. Is that not your situation? 

Technically, it should be fairly straight forward to generate the parsers from column types. Take a look at the parsers_str macro in the  source code.

**Comment by xiaodaigh** [comment]
I wrote it by hand. But I thought it would be faster to provide the types and sometimes people have to ingest data in the same CSV format month after month so it makes sense to have the functionality to capture that.

PS it ran at the same speed. Actually the first time I ran it it finished in 100s and the second time 300s, after adding the types it still ran at 300s.

I was reading one month of Fannie Mae performance data, about 500MB and 31 columns with a mix of strings, integers and floats.
